plugins/mcreator-localization/help/default/item/when_right_clicked.md

```diff
@@ -1 +1,4 @@
-The procedure is executed when a player right-clicks with this item in his hand.
\ No newline at end of file
+The procedure is executed when a player right-clicks with this item in his hand.

```

Comment:
The translation for this is :

Cette procédure est exécutée quand un joueur fait un clic droit avec cet item dans sa main.
Si vous voulez que cette procédure ne soit appelée que quand une entité fait un clic droit dans l'air avec cet item,
la procédure "Lors d'un clic droit sur le bloc" devrait toujours retourner SUCCÈS/CONSUMMÉ

Reply:
I would change "que quand une entité fait" for "que lorsque qu'une entité fait". It sounds a bit better.

Reply:
> I would change "que quand une entité fait" for "que lorsque qu'une entité fait". It sounds a bit better.

You're right so in the end it makes:

Cette procédure est exécutée quand un joueur fait un clic droit avec cet item dans sa main.
Si vous voulez que cette procédure ne soit appelée que lorsqu'une entité fait un clic droit dans l'air avec cet item,
la procédure "Lors d'un clic droit sur le bloc" devrait toujours retourner SUCCÈS/CONSUMMÉ